sampleCode/CSharp/ConsoleApp/Services/Rest.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Net;
+using System.Security.Authentication;
+using RestSharp.Authenticators;
+using RestSharp.Serializers.Json;
+
+namespace ConsoleApp.Services;
+
+/// <summary>
+/// Contains all the boilerplate to send Rest Requests, handle errors, log, and return Responses
+/// </summary>
+public static class Rest
+{
+    private static readonly RestClientOptions _restClientOptions;
+    internal static readonly RestClient _restClient;
+    private static readonly JwtAuthenticator _jwtAuthenticator;
+    private static ImplanAuthentication? _implanAuthentication;
+
+    static Rest()
+    {
+        // start with a placeholder token (to be overriden with SetAuthentication)
+        _jwtAuthenticator = new JwtAuthenticator("BAD_TOKEN");
+        // setup the defaults for all Rest requests
+        _restClientOptions = new RestClientOptions
+        {
+            // This is the base endpoint for all Implan ImpactAPI Requests
+            BaseUrl = new Uri("https://api.implan.com/int/"),
+            //BaseUrl = new Uri("https://localhost:5001/external/"),
+
```

Comment:
Likely still here due to being in WIP status, but just calling out these hard coded environment pathways.